src/ui/base/resource/resource_bundle.cc

```diff
@@ -801,6 +801,11 @@ void ResourceBundle::LoadChromeResources() {
     AddOptionalDataPackFromPath(GetResourcesPakFilePath(
         "chrome_200_percent.pak"), SCALE_FACTOR_200P);
   }
+
+#if defined(USE_CBE)
+  AddDataPackFromPath(GetResourcesPakFilePath(
+      "app_runtime_content.pak"), SCALE_FACTOR_100P);
+#endif
```

Comment:
This should not be needed. Those resources are for neva app runtime, and they are indeed already loaded in the app runtime main delegate.

So we can apply a solution similar to the one used in chromium53. Check in method GetResourcesPakFilePath, where we check if the file is in DIR_ASSETS. We had this code before that:
`#if defined(OS_WEBOS)
  if (PathService::Get(base::DIR_EXE, &path))
    return path.AppendASCII(pak_name.c_str());
#endif  // defined(OS_WEBOS)
`
We could add same code, but guarded with USE_CBE. This way, we try to find the resource first in the app path, then in cbe common path. This is important because we are installing chrome specific paks in /opt/chromium68 and CBE common paths (and app runtime) in /usr/lib/cbe, so unless we move everything to common CBE data path, we need to check both paths.

Reply:
ok. Done.